src/lib/yang/parser.lua

```diff
@@ -189,7 +204,18 @@ end
 function Parser:parse_string()
    if self:check("'") then return self:parse_qstring("'")
    elseif self:check('"') then return self:parse_qstring('"')
-   else return self:take_while("[^%s;{}\"'/]") end
+   else
+      local ret = {}
+      repeat
+         table.insert(ret, self:take_while("[^%s;{}\"'/]"))
+         if self:is_eof() then break end
+         if self:peek_n(2) == "/*" then break end
+         if self:check("/") then
```

Comment:
Can you add a test for the string `foo//bar` which I assume should parse as just `foo`.  If that works, lgtm :)